src/lib/pages/backoffice/DocumentRequest/DocumentRequestDetails/DocumentRequestSteps/DocumentRequestStepPanels/helper.js

```diff
@@ -0,0 +1,41 @@
+import _cloneDeep from 'lodash/cloneDeep';
```

Comment:
In general, I would recommend avoiding as much as possible to create utils, helpers or similar (if you did not already, `Clean Code` is a very good reading). We do have `utils` modules, but I am always trying to challenge it.
On top of that, this helper is located in a specific component, but it is meant to be very generic and reusable.

My feeling is that this kind of functions become so abstract that are not re-used in other context but only in the context/use case that they were implemented for.

What about if we go for something a little more specific? You could assume that this function is to be used in the context of prefilling forms only, for example. Then, it could be placed inside the `forms` package/module.
You could then rename it to something more specific, e.g. `mapFields`.
Happy to chat about this :)

Reply:
Yes those were my doubts too but it does  work very well for these contexts so i decided to just see what your guys' opinions are. I think having it for this specific purpose is not a bad idea at all actually to justify the abstraction layer.